Additional date support - sparql 1.2 (#163)
comunica/sparqlee: README.md

| dT             | ✓ | ✓ | ? |   |   |
| bool           | ✓ | ✓ | ? |   |   |

> ### Sparql 1.2

Reviewer: ```suggestion
### SPARQL 1.2
```

> Sparqlee looks forward and already implements some Sparql 1.2 specification functions.

Reviewer: ```suggestion
Sparqlee looks forward and already implements some SPARQL 1.2 specification functions.
```

Currently, this is restricted to the [extended date](https://github.com/w3c/sparql-12/blob/main/SEP/SEP-0002/sep-0002.md) functionality.

> Please note that the new build in ADJUST function has not been implemented due to dependencies.

Reviewer: ```suggestion
Please note that the new built-in ADJUST function has not been implemented due to dependencies.
```

> Please note that the new build-in `ADJUST` function has not been implemented due to package dependencies.

Reviewer: ```suggestion
Please note that the new built-in `ADJUST` function has not been implemented due to package dependencies.
```

> For the extended date functionality, an additional context item has been added, `implicitTimezone`.

Reviewer: ```suggestion
For the extended date functionality, an additional context item has been added: `implicitTimezone`.
```

> For the extended date functionality, an additional context item has been added, `implicitTimezone`.

Reviewer: Can we move this paragraph to the config section of the README? (and add `implicitTimezone` in the snippet there)

Reviewer: Yep

The choice was made to default to UTC time so the `implicitTimezone` does not change over time

> (soo it is not daylight saving time dependent).

Reviewer: ```suggestion
(i.e., it is not dependent on daylight saving time).
```

Author: Ah, seems like this is outdated too, we altered this to default to the timezone in `now`.
